src/types/common.d.ts

```diff
@@ -111,3 +111,11 @@ export interface ModifierContextState {
   text_line: number;
   top_text_line: number;
 }
+
+/** TODO: Move to Annotation */
+export interface AnnotationInfo {
```

Comment:
This is a different type of annotation from `annotation.js`, you can call this `ClefAnnotation` and move it back to `clef.ts`.

Reply:
Done